sdk/search/azure-search-documents/azure/search/documents/__init__.py

```diff
@@ -24,162 +24,189 @@
-    UaxUrlEmailTokenizer,
-    TokenFilter,
+    AnalyzeRequest,
+    AnalyzeResult,
     AsciiFoldingTokenFilter,
+    AzureActiveDirectoryApplicationCredentials,
+    CharFilter,
     CjkBigramTokenFilter,
+    ClassicTokenizer,
     CommonGramTokenFilter,
+    ConditionalSkill,
+    CorsOptions,
+    CustomAnalyzer,
     DictionaryDecompounderTokenFilter,
+    DistanceScoringFunction,
+    DistanceScoringParameters,
     EdgeNGramTokenFilter,
+    EdgeNGramTokenizer,
     ElisionTokenFilter,
+    EncryptionKey,
+    EntityRecognitionSkill,
+    Field,
+    FreshnessScoringFunction,
+    FreshnessScoringParameters,
+    GetIndexStatisticsResult,
+    ImageAnalysisSkill,
+    InputFieldMappingEntry,
+    Index,
     KeepTokenFilter,
+    KeyPhraseExtractionSkill,
     KeywordMarkerTokenFilter,
+    KeywordTokenizer,
+    LanguageDetectionSkill,
     LengthTokenFilter,
     LimitTokenFilter,
+    MagnitudeScoringFunction,
+    MagnitudeScoringParameters,
+    MappingCharFilter,
+    MergeSkill,
+    MicrosoftLanguageStemmingTokenizer,
+    MicrosoftLanguageTokenizer,
     NGramTokenFilter,
+    NGramTokenizer,
+    OcrSkill,
+    OutputFieldMappingEntry,
     PatternCaptureTokenFilter,
+    PatternReplaceCharFilter,
     PatternReplaceTokenFilter,
     PhoneticTokenFilter,
+    RegexFlags,
+    ScoringFunction,
+    ScoringProfile,
+    SentimentSkill,
+    ShaperSkill,
     ShingleTokenFilter,
     SnowballTokenFilter,
+    SplitSkill,
+    StandardAnalyzer,
+    StandardTokenizer,
     StemmerOverrideTokenFilter,
     StemmerTokenFilter,
+    StopAnalyzer,
     StopwordsTokenFilter,
+    Suggester,
     SynonymTokenFilter,
+    TagScoringFunction,
+    TagScoringParameters,
+    TextTranslationSkill,
+    TextWeights,
+    TokenFilter,
+    TokenInfo,
+    Tokenizer,
     TruncateTokenFilter,
+    UaxUrlEmailTokenizer,
     UniqueTokenFilter,
+    WebApiSkill,
     WordDelimiterTokenFilter,
-    CharFilter,
-    MappingCharFilter,
-    PatternReplaceCharFilter,
-    AzureActiveDirectoryApplicationCredentials,
-    EncryptionKey,
-    Index,
-    GetIndexStatisticsResult,
-    AnalyzeRequest,
-    AnalyzeResult,
-    TokenInfo,
 )
+from ._service._models import PatternAnalyzer, PatternTokenizer
+from ._version import VERSION
+
+__version__ = VERSION
+
 
 __all__ = (
-    "AutocompleteQuery",
-    "IndexAction",
-    "IndexDocumentsBatch",
-    "IndexingResult",
-    "SearchIndexClient",
-    "SearchItemPaged",
-    "SearchQuery",
-    "SearchServiceClient",
-    "SuggestQuery",
-    "odata",
-    "Field",
-    "TextWeights",
-    "ScoringFunction",
-    "DistanceScoringFunction",
-    "FreshnessScoringFunction",
-    "MagnitudeScoringFunction",
-    "TagScoringFunction",
-    "DistanceScoringParameters",
-    "FreshnessScoringParameters",
-    "MagnitudeScoringParameters",
-    "TagScoringParameters",
-    "ScoringProfile",
-    "CorsOptions",
-    "Suggester",
-    "RegexFlags",
+    "AnalyzeRequest",
+    "AnalyzeResult",
```

Comment:
Do we still want to expose AnalyzeRequest & AnalyzeResult?

Reply:
I think so? The `analyze_text` still accepts an `AnalyzeRequest` as input

Reply:
If I remember correctly, we want to take dict as the input and dict as output?

Reply:
Maybe, but this PR is about skillsets so that change if we make hot should be a separate issue/PR